grin/tests/simulnet.rs

```diff
@@ -77,6 +84,148 @@ fn basic_genesis_mine() {
 	pool.run_all_servers();
 }
 
+// Create a grin server, and a stratum server.  Simulate a few JSONRpc requests
+// and verify the results.  Validate disconnected workers and broadcasting new
+// jobs.
+#[test]
+fn basic_stratum_server() {
```

Comment:
Can you break this into its own test file? The simulnet one is getting a little abused already.

Reply:
Ya, I made this change as part of https://github.com/mimblewimble/grin/pull/937